gdnative-sys/build.rs

```diff
@@ -226,7 +226,8 @@ mod api_wrapper {
                 format_ident!("godot_gdnative_ext_nativescript_{}_{}_api_struct", maj, min)
             }
             ("PLUGINSCRIPT", 1, 0) => format_ident!("godot_gdnative_ext_pluginscript_api_struct"),
-            ("ANDROID", 1, _) => format_ident!("godot_gdnative_ext_android_api_struct"),
+            ("ANDROID", 1, 0) => format_ident!("godot_gdnative_ext_android_api_struct"),
+            ("ANDROID", 1, 1) => format_ident!("godot_gdnative_ext_android_api_struct"),
```

Comment:
As far as I can see there is only a version `1.0`, but no `1.1`. Is that correct? My concern was that when a new version is added it will cause a compilation error. This would just cause a miscompilation instead.

Reply:
There is a 1.1, see: https://github.com/GodotNativeTools/godot_headers/blob/3.2/gdnative_api.json (search for "ANDROID")

It looks like Godot 3.1 had an Android 1.0, and Godot 3.2 now has an Android 1.1 but no 1.0. You can view different tagged versions of that file to see the change.

How do we want to handle this? Should I just handle both versions like the code currently does, or should we only handle the latest version of Godot? After we decide what to do, I will at least add a comment explaining whatever we decide.

Reply:
Ahw I see. That's a bit weird, ideally @BastiaanOlij should have kept the 1.0 and added an extension with version 1.1. Now it's a bit inconsistent with how the other extensions handle it.

But it is fine, this code is okay then.

Reply:
If I remember correctly, this isn't the first time I've seen versions change like this. I guess someone could check for sure by digging through the git history, but I don't want to. :)